src/CaptureClient/GpuQueueSubmissionProcessor.cpp

```diff
@@ -412,7 +415,19 @@ std::vector<TimerInfo> GpuQueueSubmissionProcessor::ProcessGpuDebugMarkers(
       color->set_blue(static_cast<uint32_t>(completed_marker.color().blue() * 255.f));
       color->set_alpha(static_cast<uint32_t>(completed_marker.color().alpha() * 255.f));
     }
-    marker_timer.set_user_data_key(completed_marker.text_key());
+
+    // We have special handling for DXVK instrumentation that extracts the encoded group_id from
+    // the label's text. The encoding is:
+    // 'DXVK__vkFunctionName#GROUP_ID', where 'GROUP_ID' is the group id.
+    uint64_t text_key = completed_marker.text_key();
```

Comment:
Nit: Merely for readability

`const uint64_t text_key = completed_marker.text_key();`

Reply:
Done